python/nano/src/bigdl/nano/tf/keras/Model.py

```diff
@@ -21,3 +21,31 @@
 class Model(tf.keras.Model):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+
+    def quantize(self, model, calib_dataset, batch=1, val_dataset=None,
+                 metric: int = 0, backend='inc', conf=None, approach='ptsq', strategy='bayesian',
+                 accuracy_criterion=None, timeout=0, max_trials=1, **kwargs):
```

Comment:
Are `timeout=0, max_trials=1` reasonable default values?

Is `kwargs` only used for `validate`?

Reply:
`timeout=0` means no time limit, `max_trials=1` means doing quantization only once.  So the default behavior is not tuning and just get the first quantized model.  
For PTQ, it's only for `keras_model.evaluate(val_dataloader, **kwargs)`

Reply:
TF script now is still under development. It's not working now. I will talk to INC team and get back with more details.